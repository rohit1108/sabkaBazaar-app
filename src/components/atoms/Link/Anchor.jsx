import React from 'react'
const Anchor = props =>(
    <a className='anchor'>{props.text}</a>
  );

export default Anchor;