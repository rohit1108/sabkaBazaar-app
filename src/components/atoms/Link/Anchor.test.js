import renderer from 'react-test-renderer'
import Anchor from './Anchor';
import ReactDOM from 'react-dom';

test('it should have right anchor component snapshot', () => {
    const tree = renderer.create(<Anchor/>).toJSON();
    expect(tree).toMatchSnapshot();
    
  });
  it('renders the comp properly', () => {
    const divEl = document.createElement('div');
    ReactDOM.render(<Anchor />, divEl);
  });

