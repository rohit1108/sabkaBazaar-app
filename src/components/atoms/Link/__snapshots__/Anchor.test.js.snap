// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should have right anchor component 1`] = `
<a
  className="anchor"
/>
`;

exports[`it should have right anchor component snapshot 1`] = `
<a
  className="anchor"
/>
`;
